bundles/org.eclipse.equinox.coordinator/META-INF/MANIFEST.MF

```diff
@@ -2,16 +2,14 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %bundleName
 Bundle-SymbolicName: org.eclipse.equinox.coordinator
-Bundle-Version: 1.4.100.qualifier
+Bundle-Version: 1.4.200.qualifier
```

Comment:
Here I am surprised that the API version checks is not complaining that you removed API and need a major version bump.

Reply:
Me too. But the API analysis is executed.